docs/_data/generated/en/common-configuration.yml

```diff
@@ -17,15 +17,15 @@ alluxio.dora.ufs.list.status.cache.nr.dirs:
 alluxio.dora.ufs.list.status.cache.ttl:
   'The TTL of the cache of UFS list status results'
 alluxio.dora.worker.metastore.rocksdb.block.index:
-  'The block index type to be used in the RocksDB block metadata table. If unset, the RocksDB default will be used.See https://rocksdb.org/blog/2018/08/23/data-block-hash-index.html'
```

Comment:
all the files under `docs/_data/generated` are automatically generated by parsing the java code. so all of these edits will be overridden the next time they are generated. not sure what a good solution would be to convert the link strings as actual links in markdown because the markdown formatting wouldn't make sense within the code

Reply:
oh... that sucks... where is the plain text generated/parsed from? can we add the <a href="...">...</a> to wherever it's generated from or is it more complicated than that haha......

Reply:
they are all somewhere within https://github.com/Alluxio/alluxio/blob/main/dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

for example, this is part of 
```
  public static final PropertyKey DORA_WORKER_METASTORE_ROCKSDB_BLOCK_INDEX =
      enumBuilder(Name.DORA_WORKER_METASTORE_ROCKSDB_BLOCK_INDEX, DataBlockIndexType.class)
          .setDescription("The block index type to be used in the RocksDB block metadata table."
              + " If unset, the RocksDB default will be used."
              + "See https://rocksdb.org/blog/2018/08/23/data-block-hash-index.html")
          .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
          .setScope(Scope.WORKER)
          .build();
```
but i am still against fixing it in this manner. 
1. i am not sure where else the description string is used. if it's not markdown/html, it won't make sense to make this change
2. fixing the current links does not prevent the same situation in future updates and then it'll be even more confusing when some links are hyperlinks and some links are text.

Reply:
ok ya i agree. we can circle back to this later. do you want me to undo these changes or just let them get overwritten?

Reply:
i can git magic it away so you don't have to manually delete